src/devrules/notifications/dispatcher.py

```
@@ -0,0 +1,17 @@
+
+class NotificationDispatcher:
+    def __init__(self, channels: Iterable[NotificationChannel]):
+        self.channels = list(channels)
+
+    def dispatch(self, event: NotificationEvent) -> None:
+        for channel in self.channels:
+            if channel.supports(event):
+                try:
+                    channel.send(event)
+                except Exception:
+                    pass  # TODO: log the error
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)